index.js

```diff
@@ -46,13 +47,22 @@ module.exports = function(plugins, options) {
             };
 
             if (node.tag === 'style' && node.content) {
-                var styles = indentResolve([].concat(node.content).join(''), indent);
-                promise = css.process(styles, options)
-                    .then(function(result) {
-                        node.content = [indentResolve(result.css, indent)];
-                    });
+                var includesStandardType = filterType.test('text/css');
+                var hasTypeAttr = node.attrs && node.attrs.type;
+                var isFilterType = hasTypeAttr && filterType.test(node.attrs.type.trim());
+                var isUndefinedType = !hasTypeAttr || includesStandardType && /^(\s+)?$/.test(node.attrs.type);
+                var isStandardCSS = includesStandardType && (isUndefinedType || isFilterType);
+                var isOther = !isStandardCSS && isFilterType;
 
-                promises.push(promise);
+                if (isStandardCSS || isOther) {
```

Comment:
There is no need to use a custom attribute, can also be any regex as filter on the type attr, As long as there is no specific correlation between MIME types <--> (Changing) PostCSS Setup, the initial checks introduced, reduce perf and  add bloat for all other users, who didn't use/need this. Options to skip processing of certain nodes is a good addition, but please not based on an unrelated matter complicating the desired feature implementation. If I miss something in terms of 'why MIME types' and current logic is usecase critical for @stevenvachon ok, but nevertheless please separate the current filter logic (checks) from the 'normal' process then. (applying only when filter is set explicitly)